pkg/storage/sqlcommon/sqlcommon.go

```diff
@@ -326,6 +327,23 @@ func NewDBInfo(db *sql.DB, stbl sq.StatementBuilderType, sqlTime interface{}) *D
 	}
 }
 
+var (
+	emailRegex       = regexp.MustCompile(`^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$`)
+	phoneNumberRegex = regexp.MustCompile(`^(\+\d{1,2}\s?)?1?\-?\(?\d{3}\)?[\s.-]\d{3}[\s.-]\d{4}$`)
```

Comment:
If you copied these regexes from somewhere do you mind documenting a link with the source?